tests/integration/cli.py

```diff
@@ -15,21 +15,24 @@ def test_add_list_delete_projects():
 
     The project list does not need to be empty before running the test."""
     output = subprocess.check_output(['haas', 'list_projects'])
-    projects = json.loads(output)
+    projects = output.split(" ")
     assert PROJECT1 not in projects
     assert PROJECT2 not in projects
+    projects.remove('\n')  # remove \n from list
     size = len(projects)
 
     subprocess.check_call(['haas', 'project_create', PROJECT1])
     output = subprocess.check_output(['haas', 'list_projects'])
-    projects = json.loads(output)
+    projects = output.split(" ")
+    projects = map(str.strip, projects)  # strips off \n from project name
```

Comment:
Might suggest `map(lambda s: s.strip(), projects)` here. I know the json library actually hands back unicode objects, not str, and that would cause an error. This is actually fine since it comes from subprocess.check_output, which returns str, but I had to stare at it a bit. The lambda is more obviously correct regardless of which type the object is. You could also use a list comprehension:

`projects = [p.strip() for p in projects]`

Which I actually like a bit better than the lambda.